GTFO-API/Patches/GearManager_Patches.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using Gear;
+using HarmonyLib;
+using Player;
+using UnhollowerBaseLib.Attributes;
+
+namespace GTFO.API.Patches
+{
+    [HarmonyPatch(typeof(GearManager))]
+    internal class GearManager_Patches
+    {
+        [HideFromIl2Cpp]
+        public static Dictionary<InventorySlot, string[]> BotFavorites { get; set; }
```

Comment:
I'm not sure how I feel about storing persistent data like this in a patch.
My thought is that we should have a PersistentData static class in the Resources namespace where we store this.
I can merge this without changes and then reimplement it a different time

Reply:
I like it, maybe in the future there would be a storage api that could use this more. Created and moved bot favorites there.